### ISSUE TITLE
[$1000] mweb - App does not refocus on compose box on message delete using edit mode on android chrome and app

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#22949]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open the app
2. Open any report
3. Send any message
4. Edit the message and remove all text
5. Remove the text and click on green tick
6. In delete popup, click on delete and observe that app does not refocus on compose box (works fine on web chrome)

## Expected Result:
App should refocus on compose box on deleting message using edit mode as it does on web chrome

## Actual Result:
App does not refocus on compose box on deleting message using edit mode

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [x] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.41-1
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/ce9a7319-3a75-4350-b51e-3340943638aa

https://github.com/Expensify/App/assets/93399543/b8609c0a-0b08-4acb-a96d-edf148912784

**Expensify/Expensify Issue URL:**
**Issue reported by:** @dhanashree-sawant
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1689363436779099

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~010ac4c166c4d90b1e</li>
        <li>Upwork Job ID: 1681531735404843008</li>
        <li>Last Price Increase: 2023-12-02</li>
    </ul>
</details>

**Comment by tjferriss** [comment]
I'm not able to reproduce the issue on the desktop app. The app correctly refocuses on the compose box.


https://github.com/Expensify/App/assets/8292562/f076ddf8-8099-4727-8370-ffa275e11fdf

**Comment by dhanashree-sawant** [comment]
Hi @tjferriss, the issue is specifically on android chrome and android app, sorry for confusion, it is not on desktop app

**Comment by tjferriss** [comment]
Thanks @allroundexperts. I've re-opened so we can investigate.

**Comment by tjferriss** [comment]
Added external label

**Comment by tjferriss** [comment]
@aimane-chnaif can you take a look at this latest proposal when you have the time?

**Comment by tjferriss** [comment]
Waiting for proposals to be reviewed.

**Comment by tjferriss** [comment]
@aimane-chnaif can you take a look at the proposals when you have the chance?

**Comment by tjferriss** [comment]
Waiting on @aimane-chnaif to review the proposals.

**Comment by tjferriss** [comment]
Waiting on @aimane-chnaif to review proposals, Melvin.

**Comment by tjferriss** [comment]
We're waiting for proposals to be reviewed

**Comment by tjferriss** [comment]
@bernhardoj thanks for referencing the tracking issue. I agree we should put this on hold for fixing the composer.

**Comment by tjferriss** [comment]
This is on hold per https://github.com/Expensify/App/issues/15992

**Comment by tjferriss** [comment]
Still on hold.

**Comment by tjferriss** [comment]
Still on hold.

**Comment by tjferriss** [comment]
Still on hold.

**Comment by tjferriss** [comment]
Still holding

**Comment by tjferriss** [comment]
@allroundexperts is this still reproducible on Android?

**Comment by tjferriss** [comment]
Sounds good to me.

**Comment by tjferriss** [comment]
@aimane-chnaif any updates here? I'm going to move this to off hold based on your previous comment.

**Comment by tjferriss** [comment]
@aimane-chnaif any updates?

**Comment by tjferriss** [comment]
@aimane-chnaif following once more, any update on this one?

**Comment by tjferriss** [comment]
@aimane-chnaif any update here?

**Comment by tjferriss** [comment]
I tend to agree with @aimane-chnaif synopsis above. I think we can safely close this issue and revisit it down the road.
